samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -567,15 +566,15 @@ public void testCommitBehaviourWhenAsyncCommitIsEnabled() throws InterruptedExce
         try {
           firstMsgCompletionLatch.await();
           secondMsgCompletionLatch.await();
-          Thread.sleep(100);
+          Thread.sleep(500);
```

Comment:
Removed Thread sleeps from tests. Accomplishing same thing by triggering coordinator.shutdown on a particular message in task impl.